addons/external_intercom/WirelessHeadgear.hpp

```diff
@@ -0,0 +1,19 @@
+class TFAR_External_Intercom_Wireless_Headgear {
+    // Vanilla + Apex + Contact
+    class H_HelmetCrew_B;
```

Comment:
What are these classes here?
just use entry=1 like Joint Rails and some things in Arma do?